content/en/docs/concepts/scheduling-eviction/pod-priority-preemption.md

```diff
@@ -63,9 +63,10 @@ The name of a PriorityClass object must be a valid
 and it cannot be prefixed with `system-`.
 
 A PriorityClass object can have any 32-bit integer value smaller than or equal
-to 1 billion. Larger numbers are reserved for critical system Pods that should
-not normally be preempted or evicted. A cluster admin should create one
-PriorityClass object for each such mapping that they want.
+to 1 billion. This means that the range of values for a PriorityClass object is
```

Comment:
@sftim I have made the attempt to make the required changes. I'm open to any suggestions and would greatly appreciate your feedback here.